Source/SelfService/Web/applications/microservice/microservicesOverview/microservicesList/microservicesDataGrid.tsx

```diff
@@ -32,12 +33,12 @@ export const MicroservicesDataGrid = ({ application, microservices }: Microservi
 
         Promise.all(microservices.map(async microservice => {
             const microserviceInfo = getMicroserviceInfo(application, microservice);
-            const status = await getMicroserviceStatus(microservice.id, microservice.environment);
+            // const status = await getMicroserviceStatus(microservice.id, microservice.environment);
 
             return {
                 ...microservice,
                 edit: microserviceInfo,
-                phase: status[0]?.phase,
+                phase: microservice.live.phase,
```

Comment:
I believe that we don't need 'phase' here anymore :)

Reply:
We use it for the list view, so I think it's still needed :)